Content.Shared/Loadouts/LoadoutPrototype.cs

```diff
@@ -0,0 +1,90 @@
+using Content.Shared.Whitelist;
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared.Loadouts
+{
+    /// <summary>
+    ///     Describes a loadout.
+    /// </summary>
+    [Prototype("loadout")]
+    public sealed class LoadoutPrototype : IPrototype
+    {
+        [ViewVariables]
+        [IdDataField]
+        public string ID { get; } = default!;
+
+        /// <summary>
+        ///     The name of this loadout.
+        /// </summary>
+        [DataField("name")]
+        public string Name = "";
```

Comment:
Is this supposed to be required.

Reply:
I think it uses the entity name if the string isNullOrEmpty